src/components/PreviewList.js

```diff
@@ -48,12 +49,12 @@ function PreviewList({
     previewGridClasses,
     previewGridProps,
     previewType,
-    classes,
     getPreviewIcon,
-    width,
-}) {
-    const previewInside = previewType === 'inside';// extract to constants?
+}) => {
+    const previewInside = previewType === 'inside';
+    const width = useWidth();
     const cols = getCols(width, filesLimit, fileObjects.length);
+    const classes = useStyles();
```

Comment:
Keep this line

```js
const classes = useStyles();
```

as the first hook of the component.

Also I think that the `getCols` function can be converted to an hook which uses `useWidth` internally